### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 10

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#492] [Esami & Esoneri M-Z]
### Traccia

<img width="946" alt="image" src="https://github.com/user-attachments/assets/cb0a4ef8-1a20-4d34-a3d0-b2c5b1798a9f">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/c1fdbdd5-04a8-429d-8457-71609c359a54)
